src/main/kotlin/no/fdk/fdk_data_transformation_service/rdf/RDFUtils.kt

```diff
@@ -0,0 +1,40 @@
+package no.fdk.fdk_data_transformation_service.rdf
+
+import no.fdk.fdk_data_transformation_service.LOGGER
+import org.apache.jena.rdf.model.*
+import org.apache.jena.riot.Lang
+import java.io.ByteArrayOutputStream
+import java.io.StringReader
+
+const val BACKUP_BASE_URI = "http://example.com/"
+
+fun parseRDFResponse(responseBody: String, rdfLanguage: Lang, rdfSource: String?): Model? {
+    val responseModel = ModelFactory.createDefaultModel()
+
+    try {
+        responseModel.read(StringReader(responseBody), BACKUP_BASE_URI, rdfLanguage.name)
```

Comment:
Tror at dette skal også være mulig:
```suggestion
    try {
        ModelFactory.createDefaultModel().read(responseBody, BACKUP_BASE_URI, rdfLanguage.name)
```

Reply:
Hadde ikke behov for denne når jeg fjernet RDFAdapter: bb497a1